modules/processing/network.py

```diff
@@ -111,6 +113,39 @@ def __init__(self, filepath):
         self.irc_requests = []
         # Dictionary containing all the results of this processing.
         self.results = {}
+        # List containing all whitelist entries.
+        self.whitelist = self._build_whitelist()
+        # List for holding whitelisted IP-s according to DNS responses
+        self.whitelist_ips = []
+        # state of whitelisting
+        self.whitelist_enabled = self._build_whitelist_conf()
+        # List of known good DNS servers
+        self.known_dns = self._build_known_dns()
+
+    def _build_whitelist(self):
+        result = []
+        whitelist_path = os.path.join(
+            CUCKOO_ROOT, "data", "whitelist", "domain.txt"
+        )
+        for line in open(whitelist_path, 'rb'):
+            result.append(line.strip())
+        return result
+
+    def _build_whitelist_conf(self):
+        """Check if whitelisting is enabled."""
+        if not self.options.get("whitelist-dns"):
+            log.debug("Whitelisting Disabled.")
+            return False
+
+        return True
+
+    def _build_known_dns(self):
+        """Build known DNS list."""
+        _known_dns = self.options.get("allowed-dns")
+        if _known_dns is not None:
+            return _known_dns.split(",")
```

Comment:
Needs some more stripping in case the user specifies `8.8.8.8, 8.8.4.4` instead of without the whitespace.